src/libraries/System.Windows.Extensions/ref/System.Windows.Extensions.cs

```diff
@@ -12,15 +12,16 @@ public partial class SoundPlayer : System.ComponentModel.Component, System.Runti
         public SoundPlayer() { }
         public SoundPlayer(System.IO.Stream stream) { }
         protected SoundPlayer(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext context) { }
-        public SoundPlayer(string soundLocation) { }
+        public SoundPlayer(string? soundLocation) { }
         public bool IsLoadCompleted { get { throw null; } }
         public int LoadTimeout { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
         public string SoundLocation { get { throw null; } set { } }
-        public System.IO.Stream Stream { get { throw null; } set { } }
-        public object Tag { get { throw null; } set { } }
-        public event System.ComponentModel.AsyncCompletedEventHandler LoadCompleted { add { } remove { } }
-        public event System.EventHandler SoundLocationChanged { add { } remove { } }
-        public event System.EventHandler StreamChanged { add { } remove { } }
+        public System.IO.Stream? Stream { get { throw null; } set { } }
+        public object? Tag { get { throw null; } set { } }
+        public event System.ComponentModel.AsyncCompletedEventHandler? LoadCompleted { add { } remove { } }
+        public event System.EventHandler? SoundLocationChanged { add { } remove { } }
+        public event System.EventHandler? StreamChanged { add { } remove { } }
```

Comment:
GitHub won't let me comment on it, but below for X509Certificate2UI.SelectFromCollection, both title and message should be nullable.